src/infinite-scroll.js

```diff
@@ -119,12 +119,13 @@ angular.module(MODULE_NAME, [])
           const now = new Date().getTime();
           const remaining = wait - (now - previous);
           if (remaining <= 0) {
-            $interval.cancel(timeout);
+            $timeout.cancel(timeout);
             timeout = null;
             previous = now;
             func.call();
           } else if (!timeout) {
-            timeout = $interval(later, remaining, 1);
+            // A digest cycle is not required everytime the timer runs
+            timeout = $interval(later, remaining, 0, false);
```

Comment:
So that's what I thought. But when I change it to Timeout, it doesn't work correctly anymore. I looked at Lodash as I said previously, and they have that same interval there.

https://github.com/sroze/ngInfiniteScroll/pull/347#issuecomment-267805448